nx.json

```diff
@@ -54,5 +44,6 @@
       "!{projectRoot/cypress.config.{ts,js}",
       "!{projectRoot/**/*.cy.ts"
     ]
-  }
+  },
+  "nxCloudAccessToken": "ZmNlNjA0YzAtNTM1NS00MDIwLWFlMWItNWYxYzNiMjQ4N2VkfHJlYWQtb25seQ=="
```

Comment:
@AtofStryker Should this be public? I know you just moved it...

Reply:
its not supposed to be as it's [legacy configuration](https://nx.dev/ci/recipes/security/access-tokens#legacy-methods-of-setting-ci-access-tokens) but I figured addressing that was out of scope here. At some point we should see what we can clean up in our monorepo tools/processes.